docs/src/rules/no-undef-init.md

```diff
@@ -150,3 +150,20 @@ for (i = 0; i < 10; i++) {
 ```
 
 :::
+
+Secondly, this behavior can also be observed when a variable is redeclared using `var`:
+
+```js
+function foo() {
+    var x = 1;
+    console.log(x); // output: 1
+    
+    var x;
+    console.log(x); // output: 1
+
+    var x = undefined; // output: undefined
+    console.log(x);
+}
+
+foo();
+```
```

Comment:
````suggestion
```

In this case, you can avoid redeclaration by changing it to an assignment (`x = undefined;`), or use an eslint disable comment on a specific line.
````

Reply:
Done!